macros/ui/niceTables.pl

```diff
@@ -1393,6 +1425,7 @@ sub suffix {
 
 sub css {
 	my ($a, $b) = @_;
+	$a = '' unless defined $a;
```

Comment:
I prefer the syntax `$a //= '';` for this sort of thing.  It is equivalent, and much more concise.